src/tags/yaml-1.1/index.js

```diff
@@ -10,8 +10,14 @@ import { pairs } from './pairs.js'
 import { set } from './set.js'
 import { intTime, floatTime, timestamp } from './timestamp.js'
 
-const boolStringify = ({ value }) =>
-  value ? boolOptions.trueStr : boolOptions.falseStr
+const boolStringify = ({ value, sourceStr }) =>
```

Comment:
Why is the `yaml-1.1` behaviour different from `core`?

Reply:
Because we don't need to check whether the sourceStr matches the bool format in Yaml1.1 schema. THe bool format which Yaml1.1 schema supports is more than core schema.